python/ngen_cal/src/ngen/cal/ngen.py

```diff
@@ -316,13 +319,20 @@ def __init__(self, **kwargs):
                 nexus_data = self._nexus_hydro_fabric.loc[fabric['toid']]
             except KeyError:
                 raise(RuntimeError("No suitable nexus found for catchment {}".format(id)))
+            nwis = None
             try:
                 nwis = self._x_walk.loc[id.replace('cat', 'wb')]
+            except KeyError:
+                try: 
+                    nwis = self._x_walk.loc[id]
+                except KeyError:
+                    nwis = None
+            if nwis is not None:
                 #establish the hydro location for the observation nexus associated with this catchment
                 location = NWISLocation(nwis, nexus_data.name, nexus_data.geometry)
-                nexus = Nexus(nexus_data.name, location, (), id)
+                nexus = Nexus(nexus_data.poi_value, location, (), id)
```

Comment:
why the `poi_value` here.  This should be the nexus id.  The `NWISLocation` above should link to the poi...

Reply:
That's a mistake sorry. Fixing...